modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/LayoutsManagementToolbarPropsTransformer.js

```diff
@@ -45,18 +47,41 @@ export default function propsTransformer({portletNamespace, ...otherProps}) {
 		}
 	};
 
+	const exportTranslation = ({exportTranslationURL}) => {
+		const url = new URL(exportTranslationURL);
```

Comment:
I try to use `Liferay.Util.listCheckedExcept` here instead of getting the nodes directly but it is uglier because the method returns a string. 

Only little thing is that I update the `...[]` to `Array.from` to use the forEach in nodes.